docs/source/06_developers_contrib_guide/04_local_development.md

```diff
@@ -1,268 +0,0 @@
-# Testing
```

Comment:
We need this file for docs on how to do local testing and develop. Is there a better place that this should be located in the docs?

Reply:
The way I thought about this was a bit different: I had local and cloud deployment on the usage and install.
If we need a testing section, we should add one :)

I thought this was accepted with this structure when we discussed the new ToC of the docs. I'm studying for a school test atm so if you can, please just add the info there and I can try and make sense of a structure after my exam tomorrow.

The info that was on that file was transferred half to the install and half to the usage docs.